MAUI/Segmented-Control/right-to-left.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: Right To Left FlowDirection in .NET MAUI Segmented control | Syncfusion
+description: Learn about Right To Left support in Syncfusion .NET MAUI Segmented (SfSegmented) control in mobile and desktop applications from a single shared codebase.
```

Comment:
Learn about Right To Left support in Syncfusion .NET MAUI Segmented Control (SfSegmentedControl)

Reply:
description changed